NAMESPACE

```diff
@@ -502,6 +504,7 @@ export(ml_vocabulary)
 export(na.replace)
 export(print_jobj)
 export(random_string)
+export(reactiveSpark)
```

Comment:
maybe `reactiveStream` or `reactiveSparkStream` ?

Reply:
Also, we could ask @jcheng5 about this, but since the entire style of the package is `_`, perhaps it's okay to have e.g.`reactive_spark_stream()` here?